From: Philipp Rosenberger <p.rosenberger@kunbus.com>
Date: Wed, 24 Mar 2021 11:38:25 +0100
Subject: Debundel python3 paho mqtt client

Bundling 3rd party software within the package is bad. Use the
python3-paho-mqtt package instead.

Signed-off-by: Philipp Rosenberger <p.rosenberger@kunbus.com>
---
 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index 754428a..772aff7 100644
--- a/setup.py
+++ b/setup.py
@@ -49,8 +49,6 @@ setup(
         ]),
         ("share/revpipyload", glob("revpipyload/*.*")),
         ("share/revpipyload/shared", glob("revpipyload/shared/*.*")),
-        ("share/revpipyload/paho", ["lib/paho/__init__.py"]),
-        ("share/revpipyload/paho/mqtt", glob("lib/paho/mqtt/*.*")),
         ("/var/lib/revpipyload", [
             "data/var/lib/revpipyload/.placeholder",
         ])
